src/lights/roof_led/index.md

```diff
@@ -19,4 +15,26 @@ All roof LEDs are controlled by a single controller. It is connected to the same
 network as the servers and LMixer is streaming the layer data over the network
 using UDP.
 
-[roof_indexing]: ./images/roof_indexing.png "The addressing for roof LEDs"
\ No newline at end of file
+[roof_indexing]: ./images/roof_indexing.png "The addressing for roof LEDs"
+
+
+## Raspberry Pi 3b+
+Is used to control the roof LEDs. It is powered with normal a outlet and connected to the internet with a LAN cable.
+
+![rpi_image]
+
+[rpi_image]: ./images/RPI_image.jpg "The pinout of the Raspberry Pi 3b+"
```

Comment:
I think we can skip this image since it's used for the next one with markings. The empty image adds little information on this page.